rosidl_generator_py/cmake/rosidl_generator_py_get_typesupports.cmake

```diff
@@ -15,7 +15,7 @@
 macro(accumulate_typesupports)
   set(_typesupport_impl "")
   get_rmw_typesupport(_typesupport_impl ${rmw_implementation} LANGUAGE "C")
-  list(APPEND _typesupport_impls ${_typesupport_impl})
+  list_append_unique(_typesupport_impls ${_typesupport_impl})
```

Comment:
This function will require `ament_cmake_core` which is currently not exported by this package and therefore it isn't ensured that it is available to the package calling this macro.